ChiModules/KEigenvalueSolver/initialize_precursors.cc

```diff
@@ -40,7 +41,7 @@ void KEigenvalue::Solver::InitializePrecursors()
         // ----- Loop over precursors
         for (int j = 0; j < xs->num_precursors; ++j)
         {
-          int j_map = xs->precursor_map[j];
+          int j_map = precursor_map[xs_id][j];
```

Comment:
This is a narrowing conversion. Please change `int j_map` to `size_t j_map`.

Reply:
Done.